src/vendors/oasisscan.ts

```diff
@@ -54,11 +56,20 @@ export function getOasisscanAPIs(url: string | 'https://api.oasisscan.com/mainne
   }
 
   async function getTransaction({ hash }: { hash: string }) {
+    if (hash in getTransactionCacheMap) {
+      return getTransactionCacheMap[hash]
+    }
+
     const transaction = await operationsEntity.getTransaction({
       hash,
     })
 
-    const [parsedTx] = parseTransactionsList([transaction.data])
+    const [parsedTx] = parseTransactionsList([transaction.data ?? {}])
+
+    if (transaction.code === 0 && !!transaction.data) {
```

Comment:
Comment that oasisscan returns {"code":0,"data":null} for missing or unprocessed transactions